src/components/WeatherResults.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default function WeatherResults(props){
+
+  const {
+    name,
+    country,
+    weather,
+    description,
+    temp_max,
+    temp_min,
+    weatherIcon,
+    error } = props;
+
+  //removing the numbers after the '.'
+  let temperatureMax = temp_max.toString().split('.')
```

Comment:
It looks like something is up with these temperatures. It says the weather is 270 today in Brooklyn! Any ideas?

<img width="484" alt="Screenshot 2020-01-21 09 49 24" src="https://user-images.githubusercontent.com/3793148/72814845-c1c2f700-3c33-11ea-879e-9dda44ad0f62.png">

Reply:
Yes, the temperature is displayed in Kelvin.  It turns out I should be making an api call to 'openweathermap.org/data/2.5/find?q=${regionName}&units=imperial' to have it return Fahrenheit.

Or make a function to do so. This last solution would be better for scale purposes. In case I want to implement a button for the user to switch between Celsius, Fahrenheit or Kelvin.

Should I go ahead and fix it, and resubmit the pull request?

Reply:
That fix would be great to implement if you have the time! I agree with the approach of using the last function you described, to allow people to switch between scales.